trimmerdecorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0, 10] if @nameable.correct_name > 10
+  end
+end
```

Comment:
- [ ] [OPTIONAL] This is only optional because the previous reviewer skipped it, but I strongly recommend you fix it.
There is an error with your implementation of `correct_name` method in the `TrimmerDecorator` class as shown in the image below 👇 
![image](https://user-images.githubusercontent.com/46839548/182426859-dcc848e1-c953-48a7-8997-283bdfd03229.png)